serverless.yml

```diff
@@ -101,7 +101,33 @@ stepFunctions:
                 - States.ALL
                 ResultPath: "$.error-info"
                 Next: errorHandler
+            Next: tsTransformMapper
+          tsTransformMapper:
+            Type: Map
             Next: defaultSuccess
+            ItemsPath: $.timeSeries
+            MaxConcurrency: 0
+            Iterator:
+              StartAt: tsDataTypeIdentifier
+              States:
+                tsDataTypeIdentifier:
+                  Type: Pass
+                  Next: tsDataTypeRouter
+                tsDataTypeRouter:
+                  Type: Choice
+                  Choices:
+                    - Variable: $.dataType
+                      StringEquals: tsDailyValueStatisticalTransform
+                      Next: tsDailyValueStatisticalTransform
+                  Default: defaultMapSuccess
+                tsDailyValueStatisticalTransform:
+                  Type: Pass
+                  Next: loadToObservationDb
+                loadToObservationDb:
+                  Type: Pass
+                  End: true
```

Comment:
Any recommendations on the use of "End: true" vs. "Type: Succeed"?

Reply:
I've generally thinking that `Type: Succeed` is used for states that needed some sort of default the fallback on or if we wanted to specifically have the step function succeed if a particular state fails. Conversely, I've been thinking `End: true` should be used for terminal states wrap up the workflow. Not sure if that makes any sense... :ghost: